src/helm/proxy/clients/openai_client.py

```diff
@@ -187,8 +191,83 @@ def do_it():
             embedding=embedding,
         )
 
+    @staticmethod
+    def _get_tokenizer_name(tokenizer: str) -> str:
+        return tokenizer.split("/")[1]
+
     def tokenize(self, request: TokenizationRequest) -> TokenizationRequestResult:
-        raise NotImplementedError("Use the HuggingFaceClient to tokenize.")
+        # For reproducibility purposes, we use huggingface/gpt2 as the default tokenizer.
+        # except for gpt-3.5 turbo models and gpt-4 that use the tiktoken library.
+
+        if request.tokenizer != "openai/cl100k_base":
+            raise ValueError(
+                f"{request.tokenizer} is not supported."
+                + "Only openai/cl100k-base is supported by the tiktoken library for now."
```

Comment:
nit: `Only openai/cl100k-base is supported in HELM for now` (because tiktoken has other tokenizers already)